DTXCreator/Code/05.Score/CLane.cs

```diff
@@ -23,13 +23,15 @@ public enum E種別
             GtY,
             GtP,
             GtW,
-            BsV,
+			BsV,
             BsR,
             BsG,
             BsB,
             BsY,
             BsP,
-            BsW
+            BsW, 
+			GtL,			
+			BsL
```

Comment:
non-standard formatting?

Reply:
Likely due to mixture of tabs and space. It appears fine in VS 2019 so I won't further update it.